packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/UnstakeModal/UnstakeModal.tsx

```diff
@@ -35,16 +30,16 @@ export const UnstakeModal = ({ onCancel }: UnstakeModalModalProps) => {
     if (!account || status !== 'loaded') return null;
 
     return (
-        <StyledModal
-            isCancelable
+        <NewModal
```

Comment:
Move unstake button to the footer

Reply:
![Screenshot 2024-10-11 at 9 28 28](https://github.com/user-attachments/assets/51e5a523-d103-48c8-b5ad-52f685eda555)

Reply:
done, https://github.com/trezor/trezor-suite/pull/14713/commits/03dbbc780d1e2c8bcce13f33e84bcfa328df7e64